gc/stats/ScavengerStats.hpp

```diff
@@ -219,6 +216,18 @@ class MM_ScavengerStats
 		_copy_cachesize_sum += copyCacheSize;
 	}
 
+	MMINLINE void
+	countObjectSize(uintptr_t objectSize)
+	{
```

Comment:
Is this only used for the Evacuator or does the Scavenger also track this data now?

Reply:
If it is only used by the Evacuator we need to add a compile time define as this is growing the memory requirements of the Scavenger for each thread in the system

Reply:
Scavenger also records object size histograms. Scavenger and evacuator produce identical summary information, including object size histograms and performance-related metrics, at the end of each generational gc, when EVACUATOR_DEBUG_ALWAYS is defined in EvacuatorBase.hpp. 
Scavenger does not use object size stats in any other way,  but having these stats for both algorithms helps to eliminate object size differences as dependent factors contributing to observed differences in performance-related metrics when running scavenger vs evacuator benchmarking trials.

For evacuator, object size histograms may be useful in determining optimal values for some of the controllable parameters now declared as constants in MM_EvacuatorBase, eg, definition of "small object", ie an object that can be copied inside the stack. An upcoming phase of evacuator development will involve migrating some of these constants into parameters that can be set via command-line options. The object size stats will then likely be compiled in by default (ie without EVACUATAOR_DEBUG_ALWAYS).